docs/proposals/Lifelong-learning-Ops-ui.md

```diff
@@ -0,0 +1,166 @@
+- [operation and maintenance UI development](#operation-and-maintenance-UI-development)
+    - [Motivation](#motivation)
+        - [Goals](#goals)
+    - [Proposal](#proposal)
+        - [Use Cases](#use-cases)
+    - [Design Details](#design-details)
+        - [Loki](#loki)
+        - [Prometheus](#prometheus)
+        - [Grafana](#grafna)
+    - [Install](#install)
+
+# operation and maintenance UI development
+## Moaivation
+At present, Users can only access information about lifelong learning through the command line, including logs, status, and metrics.It is inconvenent and unfriendly for users. This proposal will provide a UI for metrics, log collection, status monitoring and management based on grafana. SThis allows the user to get the information by the graphical interface
+### goals
+
+- Supports metrics collection and visualization in lifelong learning
+- Support unified management and search of application logs
+- support the management and status monitoring of lifelong learning in dashboard
+## proposal
+We proposal using grafana,loki and prometheus to display the mertic,log and statues about the lifelong learnig job. The Prometheus is used to collect the mertic and status and the loki is used to collect logs.
+### use cases
+
+- Users can search log by specifying pod and keywords in the log control panel
+- User can view the history state or the mertic of any component in a sequence diagram
+- Users can view the status of any CRD (Model, DataSet, lifelongjob) by specifying the name
+## design details
+### Architecture
+![](./images/lifelong-learning-ops-architecture.png)
```

Comment:
How does this proposal work with Sedna, especially the interaction between the proposed modules and sedna modules?

Reply:
> How does this proposal work with Sedna, especially the interaction between the proposed modules and sedna modules?

I will add Sedna to the structure diagram.

Reply:
At present, Sedna modules and the interactions with the proposal are still missing

Reply:
I have modified the architecture diagram